fe1-web/src/features/rollCall/screens/__tests__/ViewSingleRollCall.test.tsx

```diff
@@ -135,6 +143,30 @@ describe('EventRollCall', () => {
         're-opened roll calls',
         testRender(mockRollCallReopened, false, ['attendee1', 'attendee2']),
       );
+      it('while closing roll call', () => {
+        mockStore.dispatch(updateRollCall(mockRollCallReopened.toState()));
+        const { getByTestId } = render(
+          <Provider store={mockStore}>
+            <FeatureContext.Provider value={contextValue}>
+              <MockNavigator
+                component={ViewSingleRollCall}
+                params={{
+                  eventId: mockRollCallReopened.id.valueOf(),
+                  isOrganizer: true,
+                  attendeePopTokens: ATTENDEES,
+                }}
+              />
+            </FeatureContext.Provider>
+          </Provider>,
+        );
+        fireEvent.press(getByTestId('roll_call_close_button'));
+        expect(requestCloseRollCall).toHaveBeenCalledTimes(1);
+        expect(requestCloseRollCall).toHaveBeenCalledWith(
+          expect.anything(),
+          expect.anything(),
+          ATTENDEES.map((attendee) => PublicKey.fromState(attendee)),
```

Comment:
this doesn't really test the absence of duplicates, does it ?

Reply:
It does since we do not use `expect.arrayContains` but give the exact array.

Reply:
I must be missing something, but I'm going to trust you (and talk later 😅 )